dotcom-rendering/scripts/webpack/webpack.config.client.js

```diff
@@ -147,6 +147,7 @@ module.exports = ({ build, sessionId }) => ({
 			  ]
 			: []),
 	],
+	externals: getExternalModules(build),
```

Comment:
These kinds of checks give me the heeby-jeebies: so hard to track down.

It may be worth having a 1-to-1 mapping of Webpack configuration files and bundles: https://github.com/guardian/dotcom-rendering/pull/8777

Reply:
Agreed! We decided we are going to review https://github.com/guardian/dotcom-rendering/pull/8777 as soon as possible!